cpp/src/Slice/Parser.h

```diff
@@ -280,6 +280,8 @@ namespace Slice
 
         /// Contains all introductory lines up to the first tag.
         [[nodiscard]] StringList overview() const;
+        /// Returns the contents of any '@remark' or '@remarks' tags.
```

Comment:
I understand the performance of this code does not matter, however this API looks gratuitously bad.

I have a bunch of `std::list<string>` fields, and I provide accessors for them. What do they look like?

1) I return copies each time: 
```cpp
[[nodiscard]] StringList remarks() const;
```

2) I return a const& to the field
```cpp
[[nodiscard]] const StringList& remarks() const noexcept { return _remarks; }
```

Why pick (1)?